js/src/api/subscriptions/personal.spec.js

```diff
@@ -87,6 +94,10 @@ describe('api/subscriptions/personal', () => {
         expect(personal.isStarted).to.be.true;
       });
 
+      it('calls parity_accountsInfo', () => {
+        expect(api._calls.accountsInfo.length).to.be.ok;
```

Comment:
Even if it's the same result, might be more obvious if we checked that the length is greater than 0